tests/test_detectionlists_actions.py

```diff
@@ -377,3 +510,164 @@ def test_handle_action_when_list_high_risk_employee_adds_response_items_to_data(
         data = action_results[0].get_data()
         assert data[0] == _MOCK_LIST_HIGH_RISK_EMPLOYEES_RESPONSE["items"][0]
         assert data[1] == _MOCK_LIST_HIGH_RISK_EMPLOYEES_RESPONSE["items"][1]
+
+    def test_handle_action_when_get_high_risk_employee_calls_get_with_expected_params(
+        self, mock_py42_with_user
+    ):
+        param = {"username": "test@example.com"}
+        connector = _create_get_hr_connector(mock_py42_with_user)
+        connector.handle_action(param)
+        mock_py42_with_user.detectionlists.high_risk_employee.get.assert_called_once_with(
+            _TEST_USER_UID
+        )
+        assert_success(connector)
+
+    def test_handle_action_when_get_high_risk_employee_adds_response_to_data(
+        self, mocker, mock_py42_with_user
+    ):
+        param = {"username": "test@example.com"}
+        mock_py42_with_user.detectionlists.high_risk_employee.get.return_value = (
+            create_mock_response(mocker, _MOCK_GET_HIGH_RISK_EMPLOYEE_RESPONSE)
+        )
+        connector = _create_get_hr_connector(mock_py42_with_user)
+        connector.handle_action(param)
+        expected_response = dict(_MOCK_GET_HIGH_RISK_EMPLOYEE_RESPONSE)
+        expected_response["riskFactors"] = [
+            {"tag": "FLIGHT_RISK"},
+            {"tag": "CONTRACT_EMPLOYEE"}
+        ]
+        assert_successful_single_data(connector, expected_response)
+
+    def test_handle_action_when_get_high_risk_employee_updates_summary(
+        self, mock_py42_with_user
+    ):
+        param = {"username": "test@example.com"}
+        connector = _create_get_hr_connector(mock_py42_with_user)
+        connector.handle_action(param)
+        assert_succesful_summary(connector, {"is_high_risk_employee": True})
+
+    def test_handle_action_when_get_high_risk_employee_and_employee_not_found_updates_summary(
+        self, mock_py42_with_user
+    ):
+        param = {"username": "test@example.com"}
+        mock_py42_with_user.detectionlists.high_risk_employee.get.side_effect = (
+            Py42NotFoundError(mock.Mock(status=404))
+        )
+        connector = _create_get_hr_connector(mock_py42_with_user)
+        connector.handle_action(param)
+        assert_succesful_summary(connector, {"is_high_risk_employee": False})
+
+    def test_handle_action_when_add_high_risk_tag_calls_add_with_expected_args(
```

Comment:
can you add an `assert_success` to this test?

Reply:
Yes, found a couple